HintList.py

```diff
@@ -487,6 +487,7 @@ def tokens_required_by_settings(world):
     'LH Underwater Item':                                          ("the #sunken treasure in a lake# is", None, 'exclude'),
     'Hideout Gerudo Membership Card':                              ("#rescuing captured carpenters# is rewarded with", None, 'exclude'),
     'Wasteland Bombchu Salesman':                                  ("a #carpet guru# sells", None, 'exclude'),
+    'GC Medigoron':                                                ("the #Medigoron# sells", None, 'exclude'),
```

Comment:
Hate to be a pedant but I believe Medigoron is the NPC's name, so the "the" is not appropriate. It's also not used for Biggoron.

Reply:
Changed formatting to match Biggoron hint.